hipcheck/src/policy_exprs/json_pointer.rs

```diff
@@ -56,6 +57,46 @@ fn lookup_json_pointer<'val>(pointer: &str, context: &'val Value) -> Result<&'va
 	}
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(tag = "format", content = "data")]
+#[serde(rename_all = "lowercase")]
+enum JSONData {
```

Comment:
Maybe a more expressive name for this enum? It's specifically types that we have custom deserialization / expr-type detection for

Reply:
I agree that the name `JSONData` is too generic. But I couldn't think of a better name at the time.

Have any ideas?

How about `AnnotatedJSONData`, `TaggedJSONData`, `TaggedValue`, or some variation thereof?

Reply:
I'm thinking `AnnotatedJSONValue`. While these two types are both represented as `String`s, and one can imagine most specially-handled data would be as well, it seems plausible to me that other JSON representations could be used.

Reply:
Fine with me, please update it and I'll approve and merge

Reply:
Renamed to `AnnotatedJSONValue`, and added detailed comments about the formats and relevant specs.